release-tools/metabuild.sh

```diff
@@ -57,6 +57,20 @@ addvar_lib() {
 	fi
 }
 
+copyvar_LIBALL_optional_mutex() {
+	var=LIBALL
+	val=$(${MAKEBSD} -v LIBALL)
+	if [ -n "${val}" ]; then
+		# Replace with pthread, if appropriate.
```

Comment:
This comment seems overly succinct.  Perhaps `Replace optional_mutex with pthread variant, if appropriate.` would convey more information?

Reply:
I went with
```
# Replace optional_mutex_normal with pthread variant,
# if appropriate.
```

I was tempted to do `Replace _normal with _pthread, if appropriate` since that would fit onto one line.